src/main/java/core/basesyntax/HelloWorld.java

```diff
@@ -1,8 +1,17 @@
 package core.basesyntax;
 
-/**
- * Feel free to remove this class and create your own.
- */
 public class HelloWorld {
+    public static final int radius = 15;
```

Comment:
Please, take a [look](https://mate-academy.github.io/jv-program-common-mistakes/java-core/abstract-class-interface/oop-advanced#all-magic-numbers-in-your-code-should-be-constants).